lib/rsgem/tasks/output.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+module RSGem
+  module Tasks
+    #
+    # Structure to bundle all messages from a task
+    #
+    OutputStruct = Struct.new(:name, :success, :warning, :error, keyword_init: true)
```

Comment:
I think I'd call it `Output` not `OutputStruct`. That's like calling a string `OutputString` and I don't think that's a common practice in the ruby world at least 👀